test-poc/base/src/test/java/org/keycloak/test/base/ManagedResourcesTest.java

```diff
@@ -21,6 +23,9 @@ public class ManagedResourcesTest {
     @TestClient
     ClientResource clientResource;
 
+    @TestUser
+    UserResource userResource;
+
     @Test
     public void testCreatedRealm() {
         Assertions.assertEquals("ManagedResourcesTest", realmResource.toRepresentation().getRealm());
```

Comment:
Do you plan to update this value: "ManagedResourcesTest" as in the GlobalManagedResourcesTest -> ManagedResourcesTest.class.getSimpleName();?

Reply:
Updated, thanks